tests/css-pseudo-4.js

```diff
@@ -87,7 +87,20 @@ export default {
 			links: {
 				dev: '#details-content-pseudo',
 			},
-			tests: ['::details-content'],
+			tests: [
+				'::details-content',
+				'::details-content::first-letter',
+				'::details-content::first-line',
+				'::details-content::before',
+				'::details-content::after',
+				'::details-content::before::marker',
+				'::details-content::after::marker',
+				'::details-content::search-text',
+				'::details-content::target-text',
+				'::details-content::spelling-error',
+				'::details-content::grammar-error',
+				'::details-content::selection',
+			],
```

Comment:
There's also a new `::highlight()` functional pseudo-element, which I think should be tested here as well. (Though we still need to add a dedicated test for it as well, so maybe it could be added in a separate PR.)

Reply:
`::details-content` is defined as an "element-backed pseudo-element". And their [definition](https://drafts.csswg.org/css-pseudo-4/#element-backed) says this:

> All pseudo-classes and pseudo-elements are syntactically allowed after an element-backed pseudo-element (such as `x-button::part(label):hover` or `x-button::part(label)::before`), just as if the pseudo-element were a type selector; but some are disallowed from matching:
>
> * The structural pseudo-classes, `:has()` pseudo-class, `:scope` pseudo-class, and `:host/:host()/:host-context()` pseudo-classes never match.
> * `::part()` never matches. (Other element-backed pseudo-elements can, however.)

So I believe we should also test for at least one pseudo-class.

Reply:
> There's also a new `::highlight()` functional pseudo-element, which I think should be tested here as well. (Though we still need to add a dedicated test for it as well, so maybe it could be added in a separate PR.)

I'll do it in a separate PR.

Reply:
> So I believe we should also test for at least one pseudo-class.

Done